src/components/attribute-picker/attribute-picker-item.js

```diff
@@ -88,7 +97,7 @@ class AttributePickerItem extends LocalizeLabsElement(LitElement) {
 		return html`
 			<div class="d2l-labs-attribute-picker-item-wrapper">
 				<div id="d2l-labs-attribute-picker-item-text" class="d2l-body-compact d2l-labs-attribute-picker-item-text">${this.text}</div>
-				<d2l-icon aria-label="${this.localize('components:attributePicker:removeValue', 'value', this.text)}" role="button" icon="d2l-tier1:close-large-thick" ?hidden="${!this.deletable}" @click="${this._onDeleteItem}"></d2l-icon>
+				<d2l-button-icon role="button" tabindex="-1" text="${this.localize('components:attributePicker:removeValue', 'value', this.text)}" icon="tier1:close-small" ?hidden="${!this.deletable}" @click="${this._onDeleteItem}"></d2l-button-icon>
```

Comment:
Hopefully `role` and `tabindex` aren't needed? `<d2l-button-icon>` isn't actually a `button`, it just wraps a native `<button>` -- so that's the element that has the button role and receives focus.

Reply:
I can probably remove the role, but tabindex is actually needed (similar to the [`tag-list-item-mixin`](https://github.com/BrightspaceUI/core/blob/main/components/tag-list/tag-list-item-mixin.js#L309)), otherwise I was noticing some weird behaviour with tabbing, especially when trying to tab backwards.